components/Hero/Methodology/index.vue

```diff
@@ -0,0 +1,65 @@
+<template>
+  <section class="border-nuha-fushia-300 border-t-2 mt-20">
+    <div class="lg:flex lg:justify-between py-20">
+      <div class="lg:w-1/2">
+        <h4>
+          {{ t('methodology.title') }}
+        </h4>
+      </div>
+      <div class="pt-5 lg:pt-0 lg:w-1/2">
+        <p>
+          {{ t('methodology.description') }}
+        </p>
+      </div>
+    </div>
+
+    <div class="cards">
+      <HeroMethodologyCard
+        v-for="(meth, i) in methodologies"
+        :key="i"
+        :title="meth.title()"
+        :content="meth.description()"
+        :image="meth.image"
+        :read-more-link="meth.readMoreLink"
+      />
+    </div>
+  </section>
+</template>
+
+<script setup lang="ts">
+  const { t } = useI18n()
+
+  const methodologies: {
+    title(): string
+    description(): string
+    image: string
+    readMoreLink: string
+  }[] = [
+    {
+      title: () => t('methodology.aboutProject'),
+      description: () => t('methodology.aboutProjectDescription'),
+      image: '/images/nuha-meth-1.svg',
+      readMoreLink: '/',
+    },
+    {
+      title: () => t('methodology.detailed'),
+      description: () => t('methodology.detailedDescription'),
+      image: '/images/nuha-meth-2.svg',
+      readMoreLink: '/',
+    },
+    {
+      title: () => t('methodology.ethical'),
+      description: () => t('methodology.ethicalDescription'),
+      image: '/images/nuha-meth-3.svg',
+      readMoreLink: '/',
+    },
+  ]
+</script>
+
+<style lang="postcss" scoped>
+  .cards {
+    @apply grid grid-cols-1 lg:grid-cols-3;
```

Comment:
I suggest making the display show 2x3 instead of 1x3 on medium devices.
Meaning, on medium to small devices show the card and text as following:
| -Image- | -text- |
| -Image- | -text- |
| -Image- | -text- |
On Smaller devices and on, iphone se, show them as implemented right now:
| -Image- |
| -text-  |
| -Image- |
| -text-  |
| -Image- |
| -text-  |